data/json/items/comestibles/dairy.json

```diff
@@ -312,6 +312,49 @@
     "vitamins": [ [ "vitC", 1 ], [ "calcium", 7 ] ],
     "fun": 3
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "milkcereal",
+    "name": { "str_sp": "cereal with milk" },
+    "weight": "258 g",
+    "color": "yellow",
+    "spoils_in": "1 d",
+    "container": "bowl_plastic",
+    "comestible_type": "DRINK",
+    "symbol": "~",
+    "quench": 5,
+    "calories": 188,
+    "description": "Some cereal with milk added to it, a traditional breakfast food in Western societies.",
+    "price": 400,
+    "price_postapoc": 25,
+    "material": [ "milk", "junk" ],
+    "volume": "250 ml",
+    "charges": 4,
+    "phase": "liquid",
+    "flags": [ "USE_EAT_VERB" ],
+    "vitamins": [ [ "calcium", 6 ], [ "iron", 42 ], [ "vitC", 1 ] ],
+    "fun": 12
+  },
+  {
+    "id": "deepfried_butter",
+    "type": "COMESTIBLE",
+    "comestible_type": "FOOD",
+    "name": { "str_sp": "deep-fried butter" },
+    "symbol": "%",
+    "quench": -4,
+    "fun": 6,
+    "description": "A snack food made of butter coated with a batter or breading and then deep-fried.  Absolutely guaranteed to clog your arteries.",
+    "price_postapoc": 150,
+    "material": [ "milk", "wheat" ],
+    "primary_material": "milk",
+    "color": "brown",
+    "spoils_in": "3 days",
+    "volume": "250 ml",
+    "weight": "70 g",
+    "charges": 4,
+    "flags": [ "EATEN_HOT" ],
+    "calories": 413
```

Comment:
```suggestion
    "calories": 413,
    "healthy": -1
```

Reply:
I've considering doing that but wasn't sure if it was necessary, as both raw butter and player-made deep-fried food (such deep-fried tripe) lack health penalty. It appears to be limited to alcohol and pre-Cataclysm processed food.